rule_action.go

```diff
@@ -284,6 +285,10 @@ var BrandingIcons = map[string]struct{}{
 	"zoom-out":           {},
 }
 
+var hashRegex = regexp.MustCompile("^[0-9a-f]{40}$")
```

Comment:
You could use short hashes as well, but I was almost certain that would conflict with branch names so this seems better.